### ISSUE TITLE
Current Version check done in AtmosphereFramework is wrong

### ISSUE THREAD

**Issue by elakito** [Atmosphere/atmosphere#1954]
When starting the server with atmosphere-2.3.0, the following invalid message is logged.

15:14:32.557 [Thread-13] INFO  o.atmosphere.cpr.AtmosphereFramework - 

	Current version of Atmosphere 2.3.0 
	Newest version of Atmosphere available 2.2.6

**Comment by elakito** [comment]
@jfarcand 
I see the following status lines are returned at the moment
ATMO21_VERSION=2.1.10
ATMO22_VERSION=2.2.6
ATMO_RELEASE_VERSION=2.3.0
...

Are you adding ATMO23_VERSION to the list or should we use ATMO_RELEASE to check the latest 2.3.x version for 2.3.x? In the latter case, when 2.4.0 or 3.0.0 gets released, the old 2.3.x code will display no longer its next 2.3.x upgradable version but the latest release version. So I think we should have ATMO23_VERSION.
what do you think?

**Comment by jfarcand** [comment]
@Agree. Please note that with 2.2.6 you will see
```java
09:27:48.253 INFO [Thread-13] o.a.c.AtmosphereFramework [AtmosphereFramework.java:1053] Latest version of Atmosphere's JavaScript Client 2.2.11
09:27:48.253 INFO [Thread-13] o.a.c.AtmosphereFramework [AtmosphereFramework.java:1060]

	Atmosphere Framework Updates
	Minor available (bugs fixes): 2.2.6
	Major available (new features): 2.3.0
```

**Comment by jfarcand** [comment]
I mean to say the issue is only with 2.3.0.

**Comment by elakito** [comment]
@jfarcand 
Could you add ATMO23_VERSION=2.3.0 to http://async-io.org/version.html?
After that, I can adjust the code to do the look up according to its minor version so that it can show its next micro version upgrade (minor upgrade) plus its next minor version upgrade (major upgrade) if available.

**Comment by elakito** [comment]
I'll be adding the similar code to master (3.0.0-SNAPSHOT) so that we won't get the same issue when releasing 3.0.0.

**Comment by elakito** [comment]
while synching all the branches, i noticed that the code needs to handle yet another case ;-(
basically we need to handle these three cases
- only a minor upgrade is available
- both a major and a minor upgrades are available
- only a major upgrade is available

i will update my patch. hopefully the last one.

**Comment by jfarcand** [comment]
@elakito Great work. Thanks a lot! Let's wait a couple of weeks and then we can release `2.3.1` and remove the `2.2.6/2.3.0` that gets displayed right now.

**Comment by elakito** [comment]
@jfarcand 
thanks. This little thing kind of bothered me, so i had to fix it ;-). 

Regarding the current AMO22 value, I think we could revert it back to 2.2.6 even before releasing 2.3.1.

Having value 2.2.6/2.3.0 may be less confusing for released 2.3.0 but probably more confusing for released  2.2.x, in particular for 2.2.6.

To see this, here are the messages we currently see.

2.2.6 ->
	Minor available (bugs fixes): 2.2.6/2.3.0
	Major available (new features): 2.3.0

2.3.0 ->
	Current version of Atmosphere 2.3.0 
	Newest version of Atmosphere available 2.2.6/2.3.0

2.2.5 ->
	Minor available (bugs fixes): 2.2.6/2.3.0
	Major available (new features): 2.3.0

2.2.7-SNAPSHOT ->
	Atmosphere Framework Updates:
	Major Update available (new features): 2.3.0

2.3.1-SNAPSHOT ->
;;; nothing is printed

If we revert ATMO22 to 2.2.6

we will instead see

2.2.6 ->
	Major available (new features): 2.3.0

2.3.0 ->
	Current version of Atmosphere 2.3.0 
	Newest version of Atmosphere available 2.2.6

2.2.5 ->
	Minor available (bugs fixes): 2.2.6
	Major available (new features): 2.3.0

regards, aki

**Comment by jfarcand** [comment]
@elakito Done. Thanks!!!
